lib/graphql/persisted_queries.rb

```diff
@@ -12,7 +12,7 @@ module GraphQL
   module PersistedQueries
     def self.use(schema_defn, options)
```

Comment:
I now receive following error
```
/lib/graphql/persisted_queries.rb:13:in `use': wrong number of arguments (given 1, expected 2) (ArgumentError)
	from /ruby/2.6.5/lib/ruby/gems/2.6.0/gems/graphql-1.10.3/lib/graphql/schema.rb:863:in `use'
	from app/graphql/crew_schema.rb:16:in `<class:CrewSchema>'
```

This probably has to do with the following

https://github.com/rmosolgo/graphql-ruby/blob/120ec5e5fd6c9d485f57479cb3861ca1c4bb4a84/lib/graphql/schema.rb#L859-L866

I suspect that your use statement should take `options` as optional so it becomes

```ruby
def self.use(schema_defn, options = {})
```

Reply:
Good point, pushed the fix